projects/1.19/assets/dimensional-pockets-ii/dimensionalpocketsii/lang/zh_cn.json

```
@@ -0,0 +1,1072 @@
+	"dimensionalpocketsii.pocket.status.no_access_remove": "只有该口袋的所有者才能将其移除",
+	"dimensionalpocketsii.pocket.status.creator_null": "创建者为[[空]]。请向本模组的作者反馈这个问题。",
+	"dimensionalpocketsii.pocket.status.action.null": "操作不可用，口袋为[[空]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.status.action.not_owner": "操作不可用，只因你并非该口袋的所有者",
+	"dimensionalpocketsii.pocket.status.null": "移动已被取消。口袋为[[空]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.status.locked": "移动已被取消。该口袋已被其所有者锁定",
+	"dimensionalpocketsii.pocket.status.trapped": "移动已被取消。该口袋的所有者已不允许你离开这里",
+	"dimensionalpocketsii.pocket.status.broken": "口袋方块有[[破损]]",
+	"dimensionalpocketsii.pocket.status.blocked": "口袋[[已被限制]]",
+	"dimensionalpocketsii.pocket.status.location_blocked": "口袋移位[[已被限制]]",
+	"dimensionalpocketsii.pocket.status.direction_unknown": "移向[[未知]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.status.colour_update": "将口袋的颜色更改为：",
+	"dimensionalpocketsii.pocket.status.cycle_side": "侧面循环：",
+	
+	"dimensionalpocketsii.pocket.desc.creator.name": "创建者：",
+	
+	"dimensionalpocketsii.pocket.direction.enter": "已进入",
+	"dimensionalpocketsii.pocket.direction.leave": "已离开",
+	"dimensionalpocketsii.pocket.direction.unknown": "未知",
+	
+	"dimensionalpocketsii.pocket.direction.pre": "你现在：",
+	"dimensionalpocketsii.pocket.direction.suff": "该口袋",
+	
+	"dimensionalpocketsii.direction.generic.pre": "成功移动：",
+	"dimensionalpocketsii.direction.generic.mid": "到：",
+	
+	"dimensionalpocketsii.pocket.shifter_core.target_unknown_custom": "无法移动至正确的目标处",
+	"dimensionalpocketsii.pocket.shifter_core.target_unknown": "无法移动至正确的目标处。放弃世界生成",
+	"dimensionalpocketsii.pocket.shifter_core.direction_unknown": "移动已被取消。移向[[未知]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.shifter_core.dimension_null": "移动已被取消。次元为[[空]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.shifter_core.server_world_null": "移动已被取消。服务器世界为[[空]]。向本模组的作者反馈这个问题罢（恼）",
+	"dimensionalpocketsii.pocket.shifter_core.same_dimension": "移动已被取消。你已经在目标次元里面了！",
+	
+	"dimensionalpocketsii.keybind.category": "次元口袋 II",
+	"dimensionalpocketsii.keybind.suit_screen": "鞘翅胸甲显示屏键",
+	"dimensionalpocketsii.keybind.suit_shift": "鞘翅胸甲移动键",
+	"dimensionalpocketsii.keybind.suit_mode_change": "鞘翅胸甲设置键",
+	"dimensionalpocketsii.keybind.suit_ender_chest": "鞘翅胸甲末影箱键",
+	
+	"dimensionalpocketsii.item.message.shifter.linked": "链接到口袋的次元变速箱：",
+	"dimensionalpocketsii.item.message.shifter.not_linked": "你还没有将变速箱链接到你的口袋呢！",
+	"dimensionalpocketsii.item.message.shifter.not_owner": "你不能将变速箱链接到该口袋，只因你并非该口袋的所有者",
+	"dimensionalpocketsii.item.message.shifter.mode_change": "模式更改：",
+	"dimensionalpocketsii.item.message.shifter.mode_change_prefix": "传送到方块：",
+	"dimensionalpocketsii.item.message.shifter.mode_change_true": "true",
+	"dimensionalpocketsii.item.message.shifter.mode_change_false": "false",
+	"dimensionalpocketsii.item.message.shifter.mode_change_suffix": "]",
+	"dimensionalpocketsii.item.message.shifter.no_energy": "你的次元变速箱能量不足！",
```

```suggestion
	"dimensionalpocketsii.item.message.shifter.no_energy": "你的次元转移器能量不足！",
```
这玩意应该不是什么手动挡的车吧
建议自查所有原文为`Shifter`和`Shift`的地方（Shift键除外）